cost/turbonomics/rightsize_databases_recommendations/aws/turbonomics_rightsize_databases_recommendations.pt

```diff
@@ -311,6 +446,87 @@ policy "pol_turbonomic_rightsize_databases" do
       field "createdTime" do
         label "Created Time"
       end
+      field "vmAge" do
+        label "VM Age"
+      end
+      field "pricingModel" do
+        label "Pricing Model"
+      end
+      field "url" do
+        label "URL"
+      end
+      field "sizenow" do
+        label "Actual Storage Amount"
```

Comment:
Use "Current" instead of "Actual". Same applies for all labels that use that word.